src/service/run-api-service.js

```diff
@@ -4,15 +4,27 @@
  *
  * The Run API Service allows you to perform common tasks around creating and updating runs, variables, and data.
  *
- * All API calls take in an "options" object as the last parameter. The options can be used to extend/override the Run API Service defaults.
+ * To use the Run API Service, instantiate it by passing in:
  *
- * Typically, you first instantiate a [Run Manager](../run-manager/) and then access the Run Service that is automatically part of the manager.
+ * * `account`: Epicenter account id (**Team ID** for team projects, **User ID** for personal projects). 
+ * * `project`: Epicenter project id.
+ * * `model`: The name of your primary model file to use in creating the run. (See more on [Writing your Model](../../../writing_your_model/).) Note that you can wait and pass this in when you use your Run Service to `create()` a run if desired.
+ *
+ * For example, 
+ *
+ *      var rs = new F.service.Run({ account: 'acme-simulations', project: 'supply-chain-game', model: 'supply_chain_game.py' });
+ *      rs.create();
+ *      rs.do('runModel');
+ *
+ * Additionally, all API calls take in an "options" object as the last parameter. The options can be used to extend/override the Run API Service defaults listed below.
+ *
+ * Typically, you first instantiate a [Run Manager](../run-manager/) and then access the Run Service that is automatically part of the manager, rather than instantiating the Run Service directly.
```

Comment:
Prefix this with saying (something like) 'Unless you're building an administrative interface to list data across multiple runs, typically.." to illustrate when you *don't* want to use the manager